module/VuFind/src/VuFind/Auth/EmailAuthenticator.php

```diff
@@ -58,7 +59,7 @@ class EmailAuthenticator implements \VuFind\I18n\Translator\TranslatorAwareInter
     /**
      * CSRF Validator
      *
-     * @var \VuFind\Validator\Csrf $csrf CSRF validator
+     * @var \VuFind\Validator\TokenCountingCsrfInterface $csrf CSRF validator
```

Comment:
Since you have a use statement for this above, you don't have to use the fully qualified class name here.